### DISCUSSION TITLE
Looking for some one that can setup the MTA for me

### DISCUSSION THREAD

**Discussion by snipecoder** [haraka/Haraka#3302] [General]
Hello
I am looking for someone who can set up the MTA for me per my requirements for payment.

**Comment by msimerson** [comment]
Did you get your setup working to your requirements?

**Comment by snipecoder** [comment]
Hi
I was able, but not able to figure out why the bounces are not getting back to the sender-path. I can see the bounce in the output of the relay logs, but not getting back to my email account. I am using directadmin.

Can you help check my setup?
